shadows/framework/src/main/java/org/robolectric/shadows/ShadowBitmap.java

```diff
@@ -899,4 +903,27 @@ void setBufferedImage(BufferedImage bufferedImage) {
   BufferedImage getBufferedImage() {
     return bufferedImage;
   }
+
+  void drawRect(RectF r, Paint paint) {
+    if (bufferedImage == null) {
+      return;
+    }
+    int[] pixels = ((DataBufferInt) bufferedImage.getRaster().getDataBuffer()).getData();
+
+    Graphics2D graphics2D = bufferedImage.createGraphics();
```

Comment:
Maybe we need set paint color to `graphics2D` before `draw`:  `graphics2D.setColor(paint.getColor());`.

Reply:
Actually, earlier i had tried it with the same statement `graphics2D.setColor(paint.getColor());`.
But, it was giving some error..
Now, I look into it once again..

Reply:
graphics2D.setColor(Color color) accepts argument of Color type, whereas paint.getColor() returns int value.

Reply:
See https://github.com/robolectric/robolectric/pull/7197#discussion_r840598262.